scripts/test_docker_local.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+set -ex
+
+cli=./tigris
+export cli
+
+CN=test-docker-local
+
+curl -sSL https://tigris.dev/cli-linux | tar -xz -C .
+
+make docker-local
+
+PORT=9981
+TMPDIR=/tmp/test_docker_local
+
+export TIGRIS_URL=localhost:$PORT
+export TIGRIS_TEST_PORT=$PORT
+export TIGRIS_CLI_TEST_FAST=1
```

Comment:
Should we document these variables somewhere? Are people expected to set them?

Reply:
People are not expected to use them. But we still need to put it in the docs somewhere. Thanks for the suggestion.